devtools/Makefile

```
@@ -47,6 +47,8 @@ DEVTOOLS_COMMON_OBJS :=				\
 	wire/channel_type_wiregen.o		\
 	wire/tlvstream.o
 
+devtools/features: $(CCAN_OBJS) common/features.o common/utils.o wire/fromwire.o wire/towire.o devtools/features.o
```

```suggestion
devtools/features: $(CCAN_OBJS) common/features.o common/utils.o wire/fromwire.o wire/towire.o common/features.o
```

maybe this should work?